typescript/infra/scripts/verify-validator.ts

```diff
@@ -104,170 +89,259 @@ class S3Wrapper {
   }
 }
 
-async function main() {
-  const {
-    a: _validatorAddress,
-    p: prospectiveBucket,
-    c: controlBucket,
-  } = await getArgs();
+class Validator {
+  private readonly controlS3BucketClient: S3Wrapper;
+  private readonly prospectiveS3BucketClient: S3Wrapper;
 
-  const cClient = new S3Wrapper(controlBucket);
-  const pClient = new S3Wrapper(prospectiveBucket);
+  // accumulators for stats
+  private extraCheckpoints!: number[];
+  private missingCheckpoints!: number[];
+  private invalidCheckpoints!: number[];
+  private modTimeDeltasS!: number[];
```

Comment:
nit: name with something more descriptive

Reply:
I am just going to document because I don't have any better ideas at this time.